src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/StorePal.cs

```diff
@@ -301,12 +301,14 @@ private static void LoadMachineStores()
             Volatile.Write(ref s_machineRootStore, rootStorePal);
         }
 
-        private static IEnumerable<T> Append<T>(IEnumerable<T> current, T addition)
+        private static IEnumerable<T> Prepend<T>(IEnumerable<T> current, T addition)
```

Comment:
Can this just use LINQ's Prepend?
https://github.com/dotnet/corefx/blob/496c93ba8dd0cac3728e53670ec0a41bfc56e619/src/System.Linq/src/System/Linq/AppendPrepend.cs#L24

Reply:
When the Append here was written this library didn't have System.Linq in its dependency graph; and the Append->Prepend flip happened as "this is the obvious change".

I _could_ change it here, but my standard habit would be to take the port as-is if there's not a technical need to change it.  (Even if the generated enumerator is less efficient, it isn't really significant since the enumerator is only walked once during the process)

Reply:
Ok, I submitted a PR to change it in master.